package_build_list_host_darwin.json

```diff
@@ -39,7 +39,7 @@
         "squish-ccr-deb557d-rev1-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/squish-ccr --platform-name Mac --package-root ../../package-system --clean",
         "astc-encoder-3.2-rev5-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/astc-encoder --platform-name Mac --package-root ../../package-system --clean",
         "DirectXShaderCompilerDxc-1.6.2112-o3de-rev1-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/DirectXShaderCompiler --platform-name Mac --package-root ../../package-system --clean",
-        "azslc-1.8.9-rev1-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/azslc --platform-name Mac --package-root ../../package-system --clean",
+        "azslc-1.8.15-rev1-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/azslc --platform-name Mac --package-root ../../package-system/azslc/temp --clean",
```

Comment:
why add a temp folder? Its not a pattern used by any other package?

Reply:
It's actually the recommended pattern. Look at `expat` library . I learned the recommended pattern from @lawsonamzn  when I was working on vulkan validation layer package for Linux. It avoids leaving lingering `azslc_<platform>` under the `package-source` folder.